src/camera-controls.js

```diff
@@ -542,8 +542,11 @@ export default class CameraControls {
 
 	setLookAt( position, target, enableTransition ) {
 
-		this._targetEnd.copy( target );
-		this._sphericalEnd.setFromVector3( _v3.subVectors( position, target ) );
+		const _position = toVector3( position );
```

Comment:
then we can reuse `_v3` which declared at the beginning 
https://github.com/yomotsu/camera-controls/blob/master/src/camera-controls.js#L21
```suggestion
		const _position = _v3.set( positionX, positionY, positionZ );
```